libs/vscode/nx-workspace/src/lib/verify-workspace.ts

```diff
@@ -18,13 +20,34 @@ export function verifyWorkspace(): {
   const workspacePath = dirname(
     WorkspaceConfigurationStore.instance.get('nxWorkspaceJsonPath', '')
   );
+
+  const workspaceJsonPath = join(workspacePath, 'workspace.json');
+  const angularJsonPath = join(workspacePath, 'angular.json');
+
+  // try and use the workspace version of nx
+  try {
+    const cachedWorkspaceJson = cacheJson(workspaceJsonPath).json;
+    if (!cachedWorkspaceJson) {
+      const workspace = getNxWorkspacePackage().readWorkspaceConfig({
+        format: 'nx',
+        path: workspacePath,
+      } as any);
+      cacheJson(workspaceJsonPath, '', workspace);
+    }
+  } catch (e) {
+    // noop - use old way
+  }
+
   try {
-    const workspaceJsonPath = join(workspacePath, 'workspace.json');
-    const angularJsonPath = join(workspacePath, 'angular.json');
     if (fileExistsSync(workspaceJsonPath)) {
       return {
         validWorkspaceJson: true,
+        // TODO(cammisuli): change all instances to use the new version - basically reverse this to the new format
```

Comment:
👍  good call on keeping this Legacy format for an initial release, def another thing to tackle, but likely all over the place